### ISSUE TITLE
Serve custom font to client that does not have it

### ISSUE THREAD

**Issue by moritzdietz** [tsl0922/ttyd#318]
Running: ttyd 1.6.0.339889e
I am using ttyd in combination with tmux to share a tmux session with colleagues.
The problem I am faced with is that I have a custom nerdfont installed that is using glyphs.
All clients connected to ttyd that don't have that font installed will have display issues - which is obvious.

Now I saw #31 but since this is a couple years old and I didn't quiet get how to serve a custom font through ttyd I thought I would ask again.

Here are two screenshots from within firefox:
![image](https://user-images.githubusercontent.com/7935536/79446319-52fcfc00-7fde-11ea-8575-3772f1a8144b.png)
![image](https://user-images.githubusercontent.com/7935536/79446364-6314db80-7fde-11ea-917c-bd439a34ce7c.png)

**Comment by tsl0922** [comment]
Serving custom resources is not supported in ttyd (#181), but you can modify the frontend code under the [html](https://github.com/tsl0922/ttyd/tree/master/html) dir and build your own version of ttyd.

**Comment by moritzdietz** [comment]
Gotcha! Ok, then I will have to read up on how to actually do that.

**Comment by tsl0922** [comment]
@moritzdietz you may want to start from editing [template.html](https://github.com/tsl0922/ttyd/blob/master/html/src/template.html).

**Comment by kevinhughes27** [comment]
I am trying to do the same thing (sharing a tmux session and ensuring the other user has my font with glyphs). I followed the recommendation in this issue and built my own html file which has been working great. I had to make a few other changes to ensure the font loaded properly as well. I opened https://github.com/tsl0922/ttyd/pull/929 with my changes and added an optional build variable which can be used to bundle a font. This streamlines this use case a bit by adding support for custom built html files without altering the main distribution much.
